ETH-Merge-Bots/Milestones-in-TTD/src/agent.spec.ts

```diff
@@ -0,0 +1,257 @@
+import { BlockEvent, HandleBlock } from "forta-agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test";
+import { provideHandleBlock } from "./agent";
+import BigNumber from "bignumber.js";
+import { createFinding, createFinalFinding } from "./findings";
+import { getNumberOfDays, getAverageBlockTime, getEstimatedNumberOfBlocksUntilMerge } from "./utils";
+
+const TERMINAL_TOTAL_DIFFICULTY = new BigNumber("10000000");

```

Comment:
I think this could be renamed to `mockTerminalTotalDifficulty` (or `MOCK_TERMINAL_TOTAL_DIFFICULTY`) since it is for testing.